tests/ci/benchmark_framework/compare_results.py

```diff
@@ -28,47 +28,60 @@ def main():
         print("Provided files must either be .json files or .csv files", file=sys.stderr)
         sys.exit(1)
 
-    # read contents of files into a dataframe in preparation for comparison
-    # note: we're assuming that the provided input is derived from the json output of the speed tool
+    # Read contents of files into a dataframe in preparation for comparison
+    # Note: we're assuming that the provided input is derived from the json output of the speed tool
     df1 = read_data(file1)
     df2 = read_data(file2)
 
-    # only compare benchmarks that appear in both of the files
+    # Only compare benchmarks that appear in both of the files
     # We need this because the speed tool at the time of writing has some tests that are disabled for OpenSSL
-    # we're using .iloc[:, 0] here because we're filtering out rows where the content of the 0th index column in one df isn't in the other
+    # We're using .iloc[:, 0] here because we're filtering out rows where the content of the 0th index column in one df isn't in the other
     # details of .iloc can be found here: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.iloc.html
     # .shape[0] represents the number of rows in the dataframe
-    # details of .shape can be found here: https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.DataFrame.shape.html
+    # Details of .shape can be found here: https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.DataFrame.shape.html
     if df1.shape[0] > df2.shape[0]:
         df1 = df1[df1.iloc[:, 0].isin(df2.iloc[:, 0])]
     elif df2.shape[0] > df1.shape[0]:
         df2 = df2[df2.iloc[:, 0].isin(df1.iloc[:, 0])]
 
-    # reset any broken indices in the dataframe from the above
+    # Reset any broken indices in the dataframe from the above
     df1.reset_index(drop=True, inplace=True)
     df2.reset_index(drop=True, inplace=True)
 
-    # rename df1 column labels
+    # Rename df1 column labels
     df1.columns = [str(col) + '.1' for col in df1.columns]
     df2.columns = [str(col) + '.2' for col in df2.columns]
 
     print(df1.columns)
 
-    # 2nd column of the dataframe represents the number of calls
+    # Setup data
     df1_numCalls = df1['numCalls.1']
     df2_numCalls = df2['numCalls.2']
+    df1_time = df1['microseconds.1']
+    df2_time = df2['microseconds.2']
+    df1_avg_time = float(df1_time) / df1_numcalls
```

Comment:
Have you tried running the script before you pushed the changes? I doubt it would work :)
`df1_numcalls` should be `df1_numCalls`.
Nit: while we are at it, what do you think about renaming `df1_numCalls` to `df1_num_calls` because in the whole file we use `_` instead of camel case...